src/client.ts

```diff
@@ -99,10 +98,10 @@ export class SkillConfigurator implements Clova.SkillConfigurator {
   /**
    * Create lambda handler for dispatching request.
    *
-   * @returns {Function}
+   * @returns
```

Comment:
Please add a return type to this function.
If there is a specific reason why no return type is included, let me know and include a reference as to why.

Reply:
I fixed it.